### ISSUE TITLE
remove outline: none 

### ISSUE THREAD

**Issue by aronblack** [negomi/react-burger-menu#263]
It would make the close button more accessible if you remove the outline: none 
in react-burger-menu.js on line 1291

**Comment by negomi** [comment]
Thanks @aronblack, agreed. There are a few accessibility issues open already, but it hasn't been something I've had time to focus on for this component yet unfortunately.
